gausium_connector/cac_examples/ActionDefinition.yaml

```diff
@@ -96,7 +100,69 @@ spec:
   confirmation:
     required: true
   lock: false
+  description: Runs a very specific cleaning task
+  group: Cleaning tasks
 ---
+# Pause the current cleaning task
+apiVersion: v0.1
+kind: ActionDefinition
+metadata:
+    id: pause-cleaning
+    scope: tag/<account_id>/<tag_id>
+spec:
+  label: Pause Cleaning Task
+  group: Cleaning tasks
+  type: RunScript
+  arguments:
+    - name: filename
+      type: string
+      value: pause_cleaning_task
```

Comment:
This requires a script to be copied to the connector host, right? If so, I'd clarify that

Reply:
Ah, it doesn't. Disregard